test/CollateralAgreement.t.sol

```diff
@@ -126,20 +136,43 @@ contract CollateralAgreementTest is DSTestPlus {
         evm.prank(arbitrator);
         agreements.settleDispute(agreementId, settlementPositions);
 
+        // Check new settlement positions
         PositionParams[] memory agreementPositions = agreements.agreementPositions(agreementId);
         assertEq(agreementPositions[0].party, bob);
         assertEq(agreementPositions[0].balance, 1.5 * 1e18);
         assertEq(agreementPositions[1].party, alice);
         assertEq(agreementPositions[1].balance, 0);
         assertEq(agreementPositions[2].balance, 0.5 * 1e18);
 
+        // Bob withdraw from agreement
         evm.prank(bob);
         agreements.withdrawFromAgreement(agreementId);
```

Comment:
Does this line actually assert that Bob withdrew from the agreement?  I would suggest creating another test function for this use-case:  `testWithdrawFromAgreement()`